### PR TITLE
Feature/tci 8  installation cleanup and Upgrades

### PR DESCRIPTION
**Pull request by sleepingmonk**

### CODE REVIEW

**Comment by ivanbueno-jcc** [comment]
@sleepingmonk 
* Keep social api modules as-is.   Stable releases of social post twitter are not stable.
image
* Revert config/config-deprep/jcc_deprep.settings.yml
* Revert config/config-deprep/system.site.yml
* Revert config/config-md/google_tag.container.madera_analytics.yml
* Revert config/config-newsroom/google_tag.container.newsroom_analytics.yml

File: composer.json [hunk 1]
```diff
@@ -178,23 +177,23 @@
         "drupal/field_group": "^3.0-rc1",
         "drupal/focal_point": "^1.0",
         "drupal/formblock": "^1.0",
-        "drupal/fullcalendar_view": "^4.5",
+        "drupal/fullcalendar_view": "^5.0.0",
         "drupal/ga_login": "^1.0@alpha",
         "drupal/geocoder": "^3.0",
-        "drupal/geolocation": "3.1.0",
+        "drupal/geolocation": "^3.1.0",
```
Comment (melwong-jcc) [inline composer.json:183]: I know that we cannot use Geolocation 3.2.0 due to a bug during edit mode so let's hope that it jumps to 3.3.0+.

https://judasdg.atlassian.net/browse/TC-288?focusedCommentId=63096

File: composer.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-    "name": "chapter-three/jcc-srl",
-    "description": "Judicial Council of California Self-Represented Litigant portal - Drupal 8",
+    "name": "chapter-three/jcc-tc",
+    "description": "Judicial Council of California Trial Courts - Drupal 8",
```
Comment (melwong-jcc) [inline composer.json:3]: Trivial detail but we should brand as despite the repo name...
"name": "judicialcouncil/jcc-base"
"description": "Judicial Council of California Web Content Management - Drupal 8"

File: composer.json (hunk 1 above)
Comment (sleepingmonk) [reply composer.json:183]: Thanks Mel.  Good catch from memory, but I've also added a patch for that issue. https://github.com/JudicialCouncilOfCalifornia/trialcourt/pull/1718/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R365

**Comment by sleepingmonk** [comment]
> @sleepingmonk
> 
> * Keep social api modules as-is.   Stable releases of social post twitter are not stable.
> 
> image
> 
> * Revert config/config-deprep/jcc_deprep.settings.yml
> * Revert config/config-deprep/system.site.yml
> * Revert config/config-md/google_tag.container.madera_analytics.yml
> * Revert config/config-newsroom/google_tag.container.newsroom_analytics.yml

@ivanbueno-jcc 

I've reverted the config files and pinned the social modules to their previous commit refs. The refs that were checked out were actually quite old.  2017 and 2018.  Not sure how those came to be the commits that were installed, but composer lock should match what it was before now.

**Review by ivanbueno-jcc — APPROVED** [review]
Looks good.  Merge to develop.

I'll merge to stage later today, after it's been refreshed with the latest master deployment at 2:30pm.

**Comment by sleepingmonk** [comment]
I'll resolve conflicts and merge locally.  There's no interface for commenting on conflicts like there is on other platforms, so I've copied here in case there is feedback.  I'm assuming keep changes from both branches in all of these cases.

config/config-md/extlink.settings.yml

```
<<<<<<< feature/TCI-8--installation-cleanup
extlink_exclude_admin_routes: false
=======
extlink_use_external_js_file: false
extlink_noreferrer: false
>>>>>>> develop
```

config/config-webservices/filter.format.basic_html.yml
```
<<<<<<< feature/TCI-8--installation-cleanup
=======
  filter_autop:
    id: filter_autop
    provider: filter
    status: false
    weight: -42
    settings: {  }
>>>>>>> develop
```

config/config-webservices/filter.format.full_html.yml
```
<<<<<<< feature/TCI-8--installation-cleanup
  filter_html_image_secure:
    id: filter_html_image_secure
    provider: filter
    status: false
    weight: -37
    settings: {  }
=======
>>>>>>> develop

...

<<<<<<< feature/TCI-8--installation-cleanup
=======
  linkit:
    id: linkit
    provider: linkit
    status: true
    weight: -48
    settings:
      title: true
>>>>>>> develop
```

**Comment by ivanbueno-jcc** [comment]
@sleepingmonk good to go.  merge conflicts look negligible.

**Pull request merged**